src/commands/images/selectSignImage.ts

```diff
@@ -0,0 +1,94 @@
+import * as vscode from 'vscode';
+import { logInToDockerCli } from '../registries/logInToDockerCli';
+import { checkCLI } from '../../utils/checkCLI';
+import { execAsync } from '../../utils/execAsync';
+
+const cliName = 'notation';
+const goToNotaryButton = 'Download Notary';
+const NotaryURL = 'https://notaryproject.dev/docs/installation/cli/';
+const errorMessage = `Notary not found: `;
+
+//sorts the key list string into an array
+async function sortOutput(theArray: any ){
```

Comment:
This function is rather unclear. Its not doing sorting but a transformation it seems. Can you add some clarifying comments?